.golangci.yml

```diff
@@ -0,0 +1,69 @@
+run:
+  build-tags: 
+    - aws
```

Comment:
Hm, with this, now we have to update the tags in 2 places :/

Reply:
yes